back-end/server.py

```diff
@@ -8,59 +9,78 @@
 app.config['CORS_HEADERS'] = 'Content-Type'
 
 
-auth = None
+username = None
 token = None
 
 
 @app.route('/')
 def index(): return render_template('index.html')
 
 
+@app.route('/logout', methods=['POST'])
+def logMeOut():
+
+        if request.method == "POST":
+                return ("logged out", 200)
+
+
 @app.route('/login', methods=['POST'])
 @cross_origin()
 def login():
     if request.method == "POST":
-        global auth
-        auth = authorize(username=request.json['username'], password=request.json['password'])
-        if not validUser(auth): abort(400)
-        print("User {} logged in".format(request.json['username']))
+
+        global username
```

Comment:
Using globals for the username and password isn't nice.  We can leave it for now, but make a note to change this.  You can change it to be a class with instance variables (properties) or just pass them back and forth between functions as needed.

Reply:
Noted.